website/content/docs/getting-started/connect-to-target.mdx

```diff
@@ -38,51 +38,57 @@ within the authorized session. When you are finished making connections, simply
 It can be annoying to keep accepting host SSH key prompts as the port changes,
 much less having to keep copying and pasting the current port. To make this
 easier, Boundary includes connect helpers that take care of this work for you.
-These take the form of `boundary connect <subcommand>`. In the following
-example, the helper will automatically execute `ssh` for you, filling in the
-local address/port, and setting an expected host ID so that future connections
-on different automatically-allocated ports don't complain about the host ID
-changing (you'll still need to accept a host key the first time):
+These are written as `boundary connect <subcommand>`. In the following example,
+the helper will automatically execute `ssh` for you, filling in the local
+address/port, and setting an expected host ID so that future connections on
+different automatically-allocated ports don't complain about the host ID
+changing (you'll still need to accept a host key the first time).
```

Comment:
This paragraph (specifically the 2nd sentence) also reads really long and dense. Maybe we could split it or remove some info?

Also, the use of the `complain` verb here is a personification and more of a colloquial use in the English lang. It probably doesn't translate well into different languages (I can vouch for at least one where it doesn't).

We could change to something like this:

"These are written as `boundary connect <subcommand>`. In the following example,
the helper will automatically execute `ssh` for you, filling in the local address and port. 
It will also set the expected host ID to avoid warnings on future connections with different
ports. You'll still need to accept a host key the first time."

^ Not fully happy with this either, but I think it illustrates what I mean.
(I think it doesn't convey the same information the original paragraph does - I suppose we'd have to understand if a simplification is worth here considering the "Getting Started" context)

Reply:
How do you feel about the change I just pushed?

> Boundary includes connect helpers that automatically accept host SSH key prompts
for you. These are written as `boundary connect <subcommand>`. 

> In the following example, the helper will automatically execute `ssh` for you,
filling in the local address and port. An expected host ID is set to avoid
warnings on future connections when a different port is allocated automatically.
A host key must still be accepted upon first connect.

Reply:
That's just waaaaaay better! _\*chef's kiss\*_

Thanks for taking a look at it 😄